cardname/lib/cardname.rb

```diff
@@ -59,7 +58,9 @@ def stringify obj
     end
 
     def banned_re
-      %r{#{ (['['] + banned_array << joint) * '\\' + ']' }}
+      Regexp.escape joint
```

Comment:
seems like if we stop supporting the banned_array, we should take it out completely.  Now it looks like we support it but we don't.